jwst/outlier_detection/utils.py

```diff
@@ -255,13 +266,23 @@ def flag_crs_in_models_with_resampling(
     scale1,
     scale2,
     backg,
+    save_blot=False,
+    make_output_path=None,
 ):
     for image in input_models:
-        flag_resampled_model_crs(image, median_data, median_wcs, snr1, snr2, scale1, scale2, backg)
+        flag_resampled_model_crs(image, median_data, median_wcs, snr1, snr2, scale1, scale2, backg, save_blot=save_blot, make_output_path=make_output_path)
 
 
 def flag_model_crs(image, blot, snr):
     cr_mask = flag_crs(image.data, image.err, blot, snr)
     # update dq array in-place
     image.dq |= cr_mask * np.uint32(DO_NOT_USE | OUTLIER)
     log.info(f"{np.count_nonzero(cr_mask)} pixels marked as outliers")
+
+
+def _make_blot_model(input_model, blot):
+    blot_model = input_model.copy()
+    blot_model.data = blot
+    blot_model.dq *= 0
+    blot_model.err *= 0.0
```

Comment:
Should the variances be set to zero here too?  Or should they all be set to None instead?

Reply:
This reverts to the old behavior.  I am happy to set it to None though if you think that would be better.

Reply:
Oh, I see - if that was the old behavior, that's fine.